src/mjolnir/graphenhancer.cc

```diff
@@ -1344,6 +1393,15 @@ void enhance(const boost::property_tree::ptree& pt,
     }
     tilebuilder.AddAccessRestrictions(access_restrictions);
 
+    // Replace turnlanes
+    if (tl_before != turn_lanes.size()) {
+      LOG_ERROR("Mismatch in turn lane count before " + std::to_string(tl_before) +
+                ""
+                " and after " +
+                std::to_string(turn_lanes.size()) + " tileid = " + std::to_string(tile_id.tileid()));
```

Comment:
What action should we take if we see this error message?

Reply:
The tile creation process usually seg faults when things like this happen.  This is just an indicator to help us figure it out.